tools/google-cloud-support-slackbot/requirements.txt

```diff
@@ -0,0 +1,56 @@
+aiohttp==3.7.4.post0
+asn1crypto==0.24.0
+async-timeout==3.0.1
+attrs==21.2.0
+cachetools==4.2.2
+certifi==2021.5.30
+chardet==4.0.0
```

Comment:
This library uses [GPL license](https://github.com/chardet/chardet/blob/master/LICENSE). Is it ok?

Reply:
I'm confirming with our opensource team if this is okay. I believe it should be since we are using the code via a package manager and not actually copying any of the source code in with the release, but I will see what guidance they have on this.

Reply:
What can we do to get rid of this dependency? I guess it's enough to update your `requests` library to `2.26.0`. See: https://github.com/psf/requests/blob/master/HISTORY.md#2260-2021-07-13
This may not be a problem for Google, but customers may have different requirements.  Some customers may also deploy applications in a different way e.g. using containers.

Reply:
Can we contribute to `aiohttp` to avoid licensing issue for that project or consult with legal team admissibility of LGPL-licensed dependencies for that project? There is `charset_normalizer` as MIT-license alternative and it worth to follow how `requests` avoided `chardet` as dependency: https://github.com/psf/requests/pull/5797

Reply:
It looks like ousret who worked on the requests PR is already on it: https://github.com/aio-libs/aiohttp/pull/5930#issue-703965067

I don't know how quickly this will go through, but for now my thoughts are we focus on getting an MVP out the door for teams that can adopt it without licensing issues so we can start getting feedback on the application. I will start up a trello or something for this though so we can continue to track it and other requests so we don't lose them

Reply:
Our open source team got back to me and the guidelines are that transitive dependencies don't need to be declared in the requirements file, and that as long as `chardet` is being called by an upstream process and not directly or being distributed by us, that we don't trigger any compliance obligations. I'll still look to help where I can on getting `aiohttp` to use `charset-normalizer` though to make sure the most number of people are capable of adopting this. For now I've removed `chardet` from the file, but I've got it noted on the trello board to follow up on this: https://trello.com/c/eRfIKquC/16-follow-up-on-chardet-dependency-removal